### ISSUE TITLE
Server auto-update

### ISSUE THREAD

**Issue by sollevix67** [mroote/factorio-server-manager#177]
Hello,

For those who have forked this repository, have you added the possibility to update the factorio binary automatically before starting the server  ?

Or by clicking on a button ?

Thanks.

**Comment by knoxfighter** [comment]
Not yet, no :(
An autoupdate with the current layout is hard to implement and will take a while. Im currently working on other factorio related projects, but when i come back to this server manager, its high on the list to implement. (After implementing the new UI and fixing all the broken stuff)

Duplicate of #10

**Comment by knoxfighter** [comment]
Implemented with #179
